.circleci/config.yml

```diff
@@ -32,13 +32,18 @@ defaults: &defaults
         path: test-reports
 
 jobs:
+  "python-3.7-symengine":
+    <<: *defaults
+    environment:
+      PIP_EXTRA_INSTALLATION: numpy symengine==0.5.0
```

Comment:
You need to set `USE_SYMENGINE` here otherwise sympy will not use it.

Reply:
Forgot it during force-push....